Products/Zuul/catalog/events.py

```diff
@@ -57,14 +57,15 @@ def _get_object_to_index(ob):
 def onIndexingEvent(ob, event):
     model_catalog = getUtility(IModelCatalog)
     object_to_index = _get_object_to_index(ob)
-
+        
     idxs = event.idxs
     if isinstance(idxs, basestring):
         idxs = [idxs]
 
     if idxs:
         s_idxs = set(idxs)
-        obj_indexes = set(model_catalog.get_indexes(object_to_index))
+        indexed, stored, _ = model_catalog.get_indexes(object_to_index)
```

Comment:
This method is returning all available indexes instead of the indexes for the specific object. We should open up a jira for this.

Reply:
https://jira.zenoss.com/browse/ZEN-29090